src/ProgressOnderwijsUtils/Data/SqlBatch.cs

```diff
@@ -143,7 +145,15 @@ public T[] Execute(SqlConnection conn)
         } catch (Exception ex) {
             throw cmd.CreateExceptionWithTextAndArguments(ex, this, "DataReaderToSingleRowUnpacker failed");
         }
-        return ParameterizedSqlObjectMapper.ReaderToArray(this, reader, unpacker, cmd);
+        var readerToArray = ParameterizedSqlObjectMapper.ReaderToArray(this, reader, unpacker, cmd);
```

Comment:
de variabele hier is ook verwarrend, evt zoiets:
```suggestion
        var rows = ParameterizedSqlObjectMapper.ReaderToArray(this, reader, unpacker, cmd);
```